### PR TITLE
Tutorial March 5 work

### PR DESCRIPTION
**Pull request by pe7r**
Added Header, ToDoList with ToDo items, in NonProject folder I was trying to work with props.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .App {
```
Comment (ruslan-palahin) [inline src/App.css]: please use lower-cased dashed names. Examples:
.app
.app-logo

File: src/components/ToDoItem.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import './ToDoItem.css'
+
+function ToDoItem() {
+	return (
+		<div className="todo-item">
+			<input type="checkbox" />
+        	<p>Task text will be here</p>
```
Comment (ruslan-palahin) [inline src/components/ToDoItem.js]: please keep valid tabulation

**Review by ruslan-palahin — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 1 above)
Comment (pe7r) [reply src/App.css]: It was made by default. Do I need to change this?

File: src/components/ToDoItem.js (hunk 2 above)
Comment (pe7r) [reply src/components/ToDoItem.js]: I'm trying to fix it, but on local computer it's all good, when i want to change and save it, git doesn't see the change so I have difficulties changing it on Github.

File: src/App.css (hunk 1 above)
Comment (ruslan-palahin) [reply src/App.css]: yes please - just to have a basic idea how to process PR rejections

File: src/components/ToDoItem.js (hunk 2 above)
Comment (ruslan-palahin) [reply src/components/ToDoItem.js]: You never have to fix it on github. 
1. fix locally
2. commmit locally
3. push to this branch
4. notify me
Thanks

Comment (pe7r) [reply src/components/ToDoItem.js]: Fixed it.

File: src/App.css (hunk 1 above)
Comment (pe7r) [reply src/App.css]: Fixed it.

**Review by ruslan-palahin — APPROVED** [review]

**Pull request merged**